mps-model-adapters/build.gradle.kts

```diff
@@ -3,14 +3,15 @@ plugins {
     `maven-publish`
 }
 
-val mpsVersion = project.findProperty("mps.version")?.toString().takeIf { !it.isNullOrBlank() } ?: "2020.3.6"
+val mpsVersion = project.findProperty("mps.version")?.toString().takeIf { !it.isNullOrBlank() } ?: "2021.1.4"
 
 dependencies {
     api(project(":model-api"))
 
-    compileOnly("com.jetbrains:mps-openapi:$mpsVersion")
-    compileOnly("com.jetbrains:mps-core:$mpsVersion")
-    compileOnly("com.jetbrains:mps-environment:$mpsVersion")
+    implementation("com.jetbrains:mps-openapi:$mpsVersion")
```

Comment:
I'm still debating if changing this to implementation was the right decision.
An alternative would be to keep this declared as compileOnly but then the user would have to add `mps-openapi` and `mps-core` buildscript and implementation dependencies in their mps version manually as it's needed for `RepositoryLanguage`, which would be unintuitive.

Reply:
The only exception is `RepositoryLanguage`. That's an object one wants to use outside of MPS. For example to register it. But it also has dependencies to these jars. `SConceptAdapterByID` and `SInterfaceConceptAdapterByID` are required to initialize `BaseConcept` and `INamedConcept`.
Could we somehow get rid of those dependencies?